install/kots/manifests/gitpod-installer-job.yaml

```diff
@@ -79,6 +79,16 @@ spec:
               fi
 
               echo "Gitpod: Generate the base Installer config"
+              echo "Gitpod: Create a Helm template directory"
```

Comment:
The `echo "Gitpod: Generate the base Installer config"` line should go above the `/app/installer init > "${CONFIG_FILE}"` so that the logging is representative of what's going on